test/active_job_object_serializer.rb

```diff
@@ -0,0 +1,32 @@
+require 'singleton'
+
+module ActiveJob
+  module Serializers
+    class ObjectSerializer
```

Comment:
Not liking that we have to redefine this class. Can we add guards on the Active Job version that's bundled, or the constant, and add Gemfiles to test with different Active Job versions?